ios/src/TGMapViewController.mm

```diff
@@ -149,6 +149,47 @@ - (void)pickFeaturesAt:(CGPoint)screenPosition
     });
 }
 
+- (void)pickLabelAt:(CGPoint)screenPosition
+{
+    if (!self.map && !self.mapViewDelegate) { return; }
```

Comment:
Agreed yeah. If that's the intended behavior that's fine. We should just document it.